client/index.jsx

```diff
@@ -41,12 +44,49 @@ class Pictures extends React.Component {
       });
   }
 
+  handleButtonClick(e) {
+    const { pictures, currPicPos } = this.state;
+    const numOfPics = pictures.length - 1;
+    const { className } = e.target;
+    const leftClick = className.baseVal === 'left-button' || (typeof className === 'string' && className.includes('left-button'));
```

Comment:
Same comment again :)  But this time baseVal has no meaning to me (an outsider reading your code) could this be named something more descriptive?  sizeBaseValue?  positionBaseValue?  etc.